Doc/library/sqlite3.rst

```diff
@@ -835,6 +835,13 @@ Exceptions
    disconnect occurs, the data source name is not found, a transaction could
    not be processed, etc.  It is a subclass of :exc:`DatabaseError`.
 
+.. exception:: NotSupportedError
+
+   Exception raised in case a method or database API was used which is not
+   supported by the database, e.g. calling the :meth:`Connection.rollback`
```

Comment:
missing "method"

Reply:
I think :meth:\`~Connection.rollback\` would be a better choice here.

Reply:
added missing "method" word & changed ```:meth:`Connection.rollback``` to the ```:meth:`~Connection.rollback```